paymentintent.go

```diff
@@ -280,12 +286,13 @@ type PaymentIntentPaymentMethodOptionsSofortParams struct {
 // PaymentIntentPaymentMethodOptionsParams represents the type-specific payment method options
 // applied to a PaymentIntent.
 type PaymentIntentPaymentMethodOptionsParams struct {
-	ACSSDebit  *PaymentIntentPaymentMethodOptionsACSSDebitParams  `form:"acss_debit"`
-	Alipay     *PaymentIntentPaymentMethodOptionsAlipayParams     `form:"alipay"`
-	Bancontact *PaymentIntentPaymentMethodOptionsBancontactParams `form:"bancontact"`
-	Card       *PaymentIntentPaymentMethodOptionsCardParams       `form:"card"`
-	OXXO       *PaymentIntentPaymentMethodOptionsOXXOParams       `form:"oxxo"`
-	Sofort     *PaymentIntentPaymentMethodOptionsSofortParams     `form:"sofort"`
+	ACSSDebit        *PaymentIntentPaymentMethodOptionsACSSDebitParams        `form:"acss_debit"`
+	Alipay           *PaymentIntentPaymentMethodOptionsAlipayParams           `form:"alipay"`
+	AfterpayClearpay *PaymentIntentPaymentMethodOptionsAfterpayClearpayParams `form:"afterpay_clearpay"`
```

Comment:
should be before Alipay in alphabetical order

Reply:
good catch